app/views/layouts/main.hbs

```diff
@@ -65,17 +70,5 @@
   </body>
 
   <footer>
-    <script src="/js/vendor/handlebars.runtime.min.js"></script>
```

Comment:
You're right -- loading this file on the client is not necessary. Early on, we tried serving templates to a front end handlebars engine, and it looks like we left in some code that grabs templates.